src/cpu/o3/rename.cc

```diff
@@ -751,6 +750,8 @@ Rename::renameInsts(ThreadID tid)
         // this instruction have been renamed.
         ppRename->notify(inst);
 
+        inst->renameEndTick = curTick() - inst->fetchTick;
```

Comment:
Do you know if that will cause any backward compatibility issues? I.e. will the Pipeline viewer and Konata still work?
I assume that the viewers simply ignore the new tick.